core/service-test-runner/pull-request-services-cli.js

```diff
@@ -8,54 +8,26 @@
 // Output:
 // travis
 // sonar
-//
-// Example:
-//
-// TRAVIS=1 TRAVIS_REPO_SLUG=badges/shields TRAVIS_PULL_REQUEST=1108 npm run test:services:pr:prepare
 
-import got from 'got'
-import { inferPullRequest } from './infer-pull-request.js'
 import servicesForTitle from './services-for-title.js'
 
-async function getTitle(owner, repo, pullRequest) {
-  const {
-    body: { title },
-  } = await got(
-    `https://api.github.com/repos/${owner}/${repo}/pulls/${pullRequest}`,
-    {
-      headers: {
-        'User-Agent': 'badges/shields',
-        Authorization: `token ${process.env.GITHUB_TOKEN}`,
-      },
-      responseType: 'json',
-    }
-  )
-  return title
-}
-
-async function main() {
-  const { owner, repo, pullRequest, slug } = inferPullRequest()
-  console.error(`PR: ${slug}`)
+let title
 
-  const title = await getTitle(owner, repo, pullRequest)
-
-  console.error(`Title: ${title}\n`)
-  const services = servicesForTitle(title)
-  if (services.length === 0) {
-    console.error('No services found. Nothing to do.')
-  } else {
-    console.error(
-      `Services: (${services.length} found) ${services.join(', ')}\n`
-    )
-    console.log(services.join('\n'))
+try {
+  if (process.argv.length < 3) {
+    throw new Error()
   }
+  title = process.argv[2]
```

Comment:
I know this is right because it works, but I'm having a little trouble convincing myself I've got the correct mental model of the flow. Is it just that this is expecting that the `...:prepare` npm script was invoked with the title arg in the first true arg position?

Reply:
Even though we invoke it with `npm run...`,
`argv[0]` is the full path to `node`
`argv[1]` is the full path to `pull-request-services-cli.js`
and then `argv[2]` is the first positional argument.

Reply:
and we call it here
https://github.com/badges/shields/blob/00fe69418eeeb963ca94f817438a1a744bc3b355/.github/actions/service-tests/action.yml#L47-L57